looper/command_models/commands.py

```diff
@@ -56,12 +116,135 @@ def create_model(self) -> Type[pydantic.BaseModel]:
         ArgumentEnum.SKIP_FILE_CHECKS.value,
         ArgumentEnum.COMPUTE.value,
         ArgumentEnum.PACKAGE.value,
-        ArgumentEnum.SETTINGS.value,
     ],
 )
+
+# TABLE
+TableParser = Command(
+    "table",
+    MESSAGE_BY_SUBCOMMAND["table"],
+    [],
+)
+
+
+# REPORT
+ReportParser = Command(
+    "report",
+    MESSAGE_BY_SUBCOMMAND["report"],
+    [
+        ArgumentEnum.PORTABLE.value,
+    ],
+)
+
+# DESTROY
+DestroyParser = Command(
+    "destroy",
+    MESSAGE_BY_SUBCOMMAND["destroy"],
+    [
+        ArgumentEnum.DRY_RUN.value,
+        ArgumentEnum.FORCE_YES.value,
+    ],
+)
+
+# CHECK
+CheckParser = Command(
+    "check",
+    MESSAGE_BY_SUBCOMMAND["check"],
+    [
+        ArgumentEnum.DESCRIBE_CODES.value,
+        ArgumentEnum.ITEMIZED.value,
+        ArgumentEnum.FLAGS.value,
+    ],
+)
+
+# CLEAN
+CleanParser = Command(
+    "clean",
+    MESSAGE_BY_SUBCOMMAND["clean"],
+    [
+        ArgumentEnum.DRY_RUN.value,
+        ArgumentEnum.FORCE_YES.value,
+    ],
+)
+
+# INSPECT
+# TODO Did this move to Eido?
```

Comment:
Older Comments in the Looper code suggest that inspect was moved to Eido. Was this functionality ever successfully migrated?

Reply:
I think eido has an inspect, which inspects a pep, but wouldn't looper inspect be inspecting something else (the looper project) ?

That said, when I run `looper inspect` on a hello_looper project, it is broken:

```
looper inspect
Looper version: 1.7.0
Command: inspect
Using looper config (.looper.yaml).
Traceback (most recent call last):
  File "/home/nsheff/.local/bin/looper", line 8, in <module>
    sys.exit(main())
  File "/home/nsheff/.local/lib/python3.8/site-packages/looper/cli_looper.py", line 760, in main
    prj.pipestat_configured_project if args.project else prj.pipestat_configured
AttributeError: 'Namespace' object has no attribute 'project'
```

Reply:
See this bug: https://github.com/pepkit/looper/issues/462

Reply:
Ok. Got it.